modules/nw-ingress-setting-a-custom-default-certificate.adoc

```diff
@@ -71,5 +71,24 @@ map[name:custom-certs-default]
 +
 The certificate secret name should match the value used to update the CR.
 
+Your Ingress Controller now uses the custom certificate, but it is not trusted
+by the web console. You must expose your CA certificate so the Ingress
+Controller can communicate with all routes.
+
+. Configure the CA certificate as the cluster proxy certificate:
++
+----
+$ oc -n openshift-config create configmap custom-ca --from-file=ca-bundle.crt=example-ca.crt
+$ oc patch proxy/cluster --type=merge --patch='{"spec":{"trustedCA":{"name":"custom-ca"}}}'
+----
++
+. Configure the certificate as the Ingress Controller's default certificate:
++
+----
+$ oc -n openshift-ingress create secret tls custom-default-cert --cert=example.crt --key=example.key
+$ oc -n openshift-ingress-operator patch ingresscontrollers/default --type=merge \
+  --patch='{"spec":{"defaultCertificate":{"name":"custom-default-cert"}}}'
+----
+
 Once the `IngressController` CR has been modified, the Ingress Operator
-will update the Ingress Controller's deployment to use the custom certificate.
+updates the Ingress Controller's deployment to use both certificates.
```

Comment:
You may want to make make reference to `$ oc get ingresscontroller/default -n openshift-ingress-operator -o yaml` to verify the status of the default ingress controller after implementing the changes.